fabtests/common/shared.c

```diff
@@ -882,7 +882,11 @@ int ft_init(void)
 		return ret;
 	}
 
-	return ft_hmem_init(opts.iface);
+	ret = ft_hmem_init(opts.iface);
```

Comment:
Didn't `ft_cuda_init` (I assume you are testing cuda) print error?

Reply:
No because I compiled fabtests without CUDA ;)